app/src/main/java/com/example/csc207courseproject/interface_adapter/login/LoginController.java

```diff
@@ -13,7 +15,8 @@ public LoginController(LoginInputBoundary loginInteractor) {
         this.loginInteractor = loginInteractor;
     }
 
-    public void execute() {
-        loginInteractor.execute();
+    public void execute(AppCompatActivity activity) {
```

Comment:
I'm pretty sure passing a view would violate CA

Reply:
Yeah by passing in the activity, then if you have to change the activity stuff you'll have to edit the input boundaries too I think